File: src/frontend/src/services/Term.js
export default class Term {
    constructor(name,termFunction) {
        this._name = name;
        this._termFunction = termFunction;
    }

    get name() {
        return this._name;
    }

    set name(value) {
        this._name = value;
    }

    get termFunction() {
        return this._termFunction;
    }

    set termFunction(value) {
        this._termFunction = value;
    }
}

File: src/main/java/cs/vsu/ru/FuzzySystem/model/Term.java
package cs.vsu.ru.FuzzySystem.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Term {
    private String name;
    private Function termFunction;
}
